### PR TITLE
Add decline payment condition endpoint

### PR DESCRIPTION
**Pull request by MeganSteenkamp**

### CODE REVIEW

File: src/api/payment-requests/payment-requests.md [hunk 1]
```diff
@@ -105,13 +105,17 @@ version (documented on this page) and the "legacy" version (documented at
...
-Conditions must be accepted for a Payment Request to be `paid` with asset types that require those conditions.
-Possible payment conditions include, among others, confirming proof of ID or confirming a promotional item was purchased.
+Some asset types require conditional approval to pay. Possible payment conditions include confirming proof of ID
+or confirming a promotional item was purchased.
+
+The `conditionsEnabled` flag should be set to true when [Creating a Payment Request](#create-a-payment-request) or these asset
+types may not be available as a [Payment Option](#payment-option).
+
+Conditions can either be [accepted](#accept-payment-condition) or [declined](#accept-payment-condition).
+Conditions must be accepted for a Payment Request to be `paid` with an asset types that requires conditional approval.
```
Comment (mark-scott) [inline src/api/payment-requests/payment-requests.md:118]:
```suggestion
Conditions must be accepted for a Payment Request to be `paid` with an asset type that requires conditional approval.
```

Comment (IsaacMurtagh) [inline src/api/payment-requests/payment-requests.md:118]: Can you add that if payment conditions are declined then the payment will be canceled?

**Review by IsaacMurtagh — APPROVED** [review]

File: src/api/payment-requests/payment-requests.md [hunk 2]
```diff
@@ -838,8 +843,7 @@ descending activity created date.
 <a name="accept-payment-condition"></a>
 ### Accept Payment Condition For a Payment Request **EXPERIMENTAL**
```
Comment (mark-scott) [inline src/api/payment-requests/payment-requests.md:844]:
```suggestion
### Accept Payment Condition for a Payment Request **EXPERIMENTAL**
```

File: src/api/payment-requests/payment-requests.md [hunk 3]
```diff
@@ -878,6 +882,48 @@ that has status `awaiting-merchant`. Returns a [Payment Activity][] in the respo
 | 403    | {% break _ MERCHANT_NOT_AUTHORIZED %} | The Payment Condition is `awaiting-patron`, therefore the merchant is not authorized to accept the condition. |
 | 403    | {% break _ CONDITION_ALREADY_SET %}   | The Payment Condition has already been accepted or declined.                                                  |
 
+<a name="decline-payment-condition"></a>
+### Decline Payment Condition For a Payment Request **EXPERIMENTAL**
```
Comment (mark-scott) [inline src/api/payment-requests/payment-requests.md:886]:
```suggestion
### Decline Payment Condition for a Payment Request **EXPERIMENTAL**
```

**Review by mark-scott — DISMISSED** [review]

File: src/api/payment-requests/payment-requests.md (hunk 1 above)
Comment (MeganSteenkamp) [reply src/api/payment-requests/payment-requests.md]: Done ✅

**Pull request merged**